src/LiveComponent/assets/src/Component/plugins/ErrorPlugin.ts

```diff
@@ -0,0 +1,224 @@
+                throw new Error(
+                    `Element "${element.nodeName}" is not supported for ${ErrorPlugin.errorAttribute} directives. Only HTMLElement and SVGElement are supported.`
+                );
+            }
+
+            // use "show" if the attribute is empty
+            const directives = parseDirectives(element.getAttribute(ErrorPlugin.errorAttribute) || 'show');
+
+            directives.forEach((directive) => {
+                if (directive.modifiers.length > 0) {
+                    // TODO: Use the `modifier.toString()` method once it's implemented
+                    throw new Error(
+                        `Modifiers are not supported for ${ErrorPlugin.errorAttribute} directives, but the following were found: "{${directive.modifiers.map((modifier) => `${modifier.name}: ${modifier.value}}`).join(', ')}" for element with tag "${element.nodeName}".`
+                    );
+                }
+            });
+
+            errorDirectives.push({
+                element,
+                directives,
+            });
+        });
+
+        return errorDirectives;
+    }
+
+    /**
+     * Parses the action and returns the opposite action if the element is not in an error state.
+     */
+    private parseErrorAction(action: string, isError: boolean) {
+        switch (action) {
+            case ErrorPlugin.supportedActions.show:
+                return isError ? 'show' : 'hide';
+
+            case ErrorPlugin.supportedActions.hide:
+                return isError ? 'hide' : 'show';
+
+            case ErrorPlugin.supportedActions.addClass:
+                return isError ? 'addClass' : 'removeClass';
+
+            case ErrorPlugin.supportedActions.removeClass:
+                return isError ? 'removeClass' : 'addClass';
+
+            case ErrorPlugin.supportedActions.addAttribute:
+                return isError ? 'addAttribute' : 'removeAttribute';
+
+            case ErrorPlugin.supportedActions.removeAttribute:
+                return isError ? 'removeAttribute' : 'addAttribute';
+
+            default:
+                throw new Error(
+                    `Unknown ${ErrorPlugin.errorAttribute} action "${action}". Supported actions are: ${Object.values(`"${ErrorPlugin.supportedActions}"`).join(', ')}.`
+                );
+        }
+    }
+
+    // Utils
+    private showElement(element: HTMLElement | SVGElement) {
+        element.style.display = 'revert';
+    }
+
+    private hideElement(element: HTMLElement | SVGElement) {
+        element.style.display = 'none';
+    }
+
+    private addClass(element: HTMLElement | SVGElement, classes: string[]) {
+        element.classList.add(...combineSpacedArray(classes));
+    }
+
+    private removeClass(element: HTMLElement | SVGElement, classes: string[]) {
+        element.classList.remove(...combineSpacedArray(classes));
+
+        if (element.classList.length === 0) {
+            element.removeAttribute('class');
+        }
```

Comment:
I think this is not necessary

Reply:
In fact i believe it is, or the next morph will consider this a change .. or let's say i _think_ i remember this

Reply:
I'm not sure if this behavior changed with the switch to Idiomorph though